pkg/autoscaler/config/config.go

```diff
@@ -155,15 +172,19 @@ func NewConfigFromMap(data map[string]string) (*Config, error) {
 
 func validate(lc *Config) (*Config, error) {
 	if lc.ScaleToZeroGracePeriod < autoscaling.WindowMin {
-		return nil, fmt.Errorf("scale-to-zero-grace-period must be at least %v, got %v", autoscaling.WindowMin, lc.ScaleToZeroGracePeriod)
+		return nil, fmt.Errorf("scale-to-zero-grace-period must be at least %v, was: %v", autoscaling.WindowMin, lc.ScaleToZeroGracePeriod)
+	}
+
+	if lc.ScaleDownDelay < 0 {
```

Comment:
Should we check it's rounded to 2s as well?

Reply:
Or at least 1s granularity?

Reply:
yeah good idea, I'll add

Reply:
(done)